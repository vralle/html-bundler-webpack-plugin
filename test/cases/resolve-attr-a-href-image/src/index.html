<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Hello World!</h1>
    <a href="index.html">home</a> | <a href="about.html">about</a>
    <a
      href='@images/kiwi.png'
      data-image-width='160'
      data-image-height='130'
      target='_blank'
      >
     <img src='@images/image.webp' width='100' alt='cat' />
    </a>
  </body>
</html>