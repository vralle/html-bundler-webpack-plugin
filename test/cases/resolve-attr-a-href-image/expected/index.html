<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Hello World!</h1>
    <a href="index.html">home</a> | <a href="about.html">about</a>
    <a
      href='img/kiwi.da3e3cc9.png'
      data-image-width='160'
      data-image-height='130'
      target='_blank'
      >
     <img src='img/image.d4711676.webp' width='100' alt='cat' />
    </a>
  </body>
</html>